### DISCUSSION TITLE
gitbook.gitbook-vscode VSCode extension lists non-existing repository

### DISCUSSION THREAD

**Discussion by kkm000** [GitbookIO/community#357] [0. General bugs and questions] [answered]
Go to https://marketplace.visualstudio.com/items?itemName=GitBook.gitbook-vscode, click on "Repository" or "Project details" subheading with the Github icon in the right column (both pointing to https://github.com/GitBookIO/gitbook-vscode-sidekick), get a 404. Probably not intended?

**Comment by addisonschultz** [comment, marked as answer]
Hey @kkm000! Nice find - It's something we're actually still building at the moment! We're testing out some new functionality and doing some internal testing at the moment. That being said, if you're interested in using something between GitBook and VS Code, would you mind sending me an email to [addison @ gitbook.com]?
